app/src/main/java/com/example/atheneum/activities/BookInfoActivity.java

```diff
@@ -0,0 +1,143 @@
+ */
+
+package com.example.atheneum.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import com.example.atheneum.R;
+import com.example.atheneum.models.Book;
+import com.example.atheneum.models.User;
+import com.example.atheneum.utils.OwnerBooksAdapter;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.ValueEventListener;
+
+public class BookInfoActivity extends AppCompatActivity {
+
+    private static final int BOOK_INFO_DELETED =0;
+    private static final int BOOK_INFO_EDITED = 1;
+
+
+    private User owner;
+    private Book book;
+    String title;
+    String author;
+    long isbn;
+    String desc;
+    private String bookID;
+
+    private TextView textTitle;
+    private TextView textAuthor;
+    private TextView textIsbn;
+    private TextView textDesc;
+    private TextView textStatus;
+
+    private Button deleteBtn;
+    private Button editBtn;
+
+    private static final String TAG = "viewBookInfo";
+
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode, @Nullable Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_book_info);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+
+        bookID = getIntent().getStringExtra("bookID");
+        Log.i("bookid value: ", bookID);
+        textTitle = (TextView) findViewById(R.id.bookTitle);
+        textAuthor = (TextView) findViewById(R.id.bookAuthor);
+        textIsbn = (TextView) findViewById(R.id.bookISBN);
+        textDesc = (TextView) findViewById(R.id.bookDescription);
+        textStatus = (TextView) findViewById(R.id.bookStatus);
+
+        deleteBtn = findViewById(R.id.buttonDelete);
+        deleteBtn.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                deleteBook();
+            }
+        });
+        editBtn = findViewById(R.id.buttonEdit);
+        //TODO: implement edit book button
+
+        retrieveBookInfo();
+
+    }
+
+    public void retrieveBookInfo(){
+        FirebaseUser firebaseUser = FirebaseAuth.getInstance().getCurrentUser();
```

Comment:
I think this is okay for now but we should eventually start putting the DB logic into the ViewModels instead of the activites and just observe the LiveData instead. It becomes to hard to change the database structure since the database is so coupled with the activity. But since I'm going to change the ownerCollection table after you merge this PR, I can do that refactor.